markdown/events/2022/hackathon-march-2022.md

```diff
@@ -28,6 +30,75 @@ Registration is now OPEN! Please sign up [here](https://scilifelab.typeform.com/
 Please check back here for more details about the event and keep an eye out on twitter and on Slack,
 but in the mean time - book the dates in your calendar!
 
+# Topic: Documentation
+
+The past twelve months have brought some major changes to the nf-core community, with the migration to Nextflow DSL2.
+The nf-core website and documentation is a cruicial part of the community.
+In this hackathon, the focus will be _all things documentation_.
+
+Generally speaking, tasks during the hackathon will typically involve going through tutorials,
+testing each command making srue they work, checking phrasing and descriptions to ensure they are clear.
+
+> Most nf-core documentation is written in _Markdown_.
+> If you're not yet familiar with Markdown, please take a few minutes to work through this tutorial: <https://www.markdowntutorial.com/>
+
+To keep things manageable, we will organise into the groups.
+You are free to change groups as you wish during the hackathon.
+
+## Group 1 - Pipeline documentation
+
+💬 Slack group: [`hackathon-march2022-pipelines`](https://nfcore.slack.com/archives/C0321U2D16Z)<br>
```

Comment:
can we have the proper chan name in the link too?

Reply:
Done 👍🏻